standard_reports/item-usage/latest/convert_data_js/src/utils.test.ts

```diff
@@ -1,158 +0,0 @@
-import {
-  calculateQuantity,
-  calculateStatValue,
-  processItemLines,
-} from "./utils";
-import inputData from "../input.json" assert { type: "json" };
-import outputData from "../output.json" assert { type: "json" };
-
-describe("test item lines", () => {
-  it("end to end item-usage", () => {
-    const result = processItemLines(inputData);
-    expect(result).toEqual(outputData.items.nodes);
-  });
-});
-
-describe("Adds monthlyConsumption correctly from query result", () => {
```

Comment:
As above, I think these unit tests have value, don't see a reason to delete them?